app/assets/stylesheets/tabs.css

```diff
@@ -120,25 +120,25 @@
 }
 
 #hot100Genre {
-  background-image: url('Hot100.png');
+  background-image: url('/Hot100.png');
 }
 
 #alternativeGenre {
-  background-image: url('Alternative.png');
+  background-image: url('/Alternative.png');
 }
 
 #hiphopGenre {
-  background-image: url('HipHop.png');
+  background-image: url('/HipHop.png');
```

Comment:
Don't think you need the slashes here, can you try removing it and see if it works